### PR TITLE
Updated CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by devlead**

### CODE REVIEW

**Comment by patriksvensson** [comment]
Looks good to me :+1:

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,42 +1,172 @@
...
+* The contribution is your own original work.
+* You have the right to assign the copyright for the work (it is not owned by your employer, or
+  you have been given copyright assignment in writing).
+* You [license](https://github.com/cake-build/cake/blob/develop/LICENSE) the contribution under the terms applied to the rest of the Cake project.
+* You agree to follow the [code of conduct](https://github.com/cake-build/cake/blob/develop/CODEOFCONDUCT.md).
 
-Start by either suggest a feature in the issue list that you want to see in Cake, or assign yourself to an existing feature/bug. Make sure that no one else is working on it already though. It's also appreciated to have some kind of discussion about the issue if it's not an easy fix.
+## Definition of trivial contributions
+It's hard to define what is a trivial contribution. Sometimes even a 1 character change can be considered significant.
+Unfortunately because it can be subjective, the decision on what is trivial comes from the committers of the project
```
Comment (daveaglick) [inline CONTRIBUTING.md]: I would suggest "maintainers of the project" over "committers of the project". The term "committers" seems ambiguous...

Comment (gep13) [inline CONTRIBUTING.md]: @daveaglick I agree with this.  @devlead can you fix up, and I will pull in, or if you are not in a position to do that, I can fix after merging.

Comment (devlead) [inline CONTRIBUTING.md]: @gep13 updated and rebased.

**Pull request merged**